static/app/views/replays/detail/console/format.tsx

```diff
@@ -92,7 +96,30 @@ export default function Format({onExpand, expandPaths, args}: FormatProps) {
     }
   });
 
-  pieces.push(str);
+  if (styling) {
+    const tempEl = document.createElement('div');
+    tempEl.setAttribute('style', styling);
```

Comment:
Not sure if there's a simpler way - I wanted an explicit list of CSS properties to allow as to avoid any XSS shenanigans.

Reply:
is `styling` a plain object or something else?

if it's a plain object should try to skip passing it into `setAttribute`. Maybe it can be splatted out to grab those 3 rules we care about.

just testing and this would work and be simpler imo:
```
const {color, 'font-weight': fontWeight, 'font-style': fontStyle} = obj;
console.log({color, fontWeight, fontStyle});

Reply:
It's a string that we turn into an object, filter via include list of properties, then convert to a React.CSSProperties object